src/main/java/creativei/enums/ExceptionType.java

```diff
@@ -8,7 +8,8 @@
     DUPLICATE_VALUE(1002, "Value already exists"),
     DATABASE_EXCEPTION(1003, "Exception occured while adding values to database"),
     INVALID_METHOD_PARAM(1004,"Invalid Method Request "),
-    DATA_NOT_AVAILABLE(1005,"No record found for requested parameter");
+    DATA_NOT_AVAILABLE(1005,"No record found for requested parameter"),
+    RESULTSET_GENERATION_EXCEPTION(1006,"ResultSet Can not be Generated");
```

Comment:
This doesn't seem to be a good exception message. Share the reason of failure in user readable text and log the actual exception.